src/main/java/com/sngular/kloadgen/sampler/schemaregistry/schema/ApicurioParsedSchema.java

```diff
@@ -1,17 +1,31 @@
 package com.sngular.kloadgen.sampler.schemaregistry.schema;
 
+import com.sngular.kloadgen.sampler.schemaregistry.adapter.impl.ApicurioParsedSchemaMetadata;
+import com.sngular.kloadgen.sampler.schemaregistry.adapter.impl.ParsedSchemaAdapter;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
 @Getter
 @Setter
 @NoArgsConstructor
-public class ApicurioParsedSchema {
+public class ApicurioParsedSchema extends ParsedSchemaAdapter {
 
   private String type;
 
   private Object schema;
 
   private String rawSchema;
+
+  @SuppressWarnings("UnnecessarilyFullyQualified")
+  private ApicurioParsedSchema(final ApicurioParsedSchema apicurioParsedSchema){
```

Comment:
Why does the constructor need an ApicurioParsedSchema argument to instantiate an object of the same type?

Reply:
changed